credentials/credstore.go

```diff
@@ -0,0 +1,81 @@
+package credentials
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/cnabio/cnab-go/utils/crud"
+)
+
+const ItemType = "credentials"
```

Comment:
Who is supposed to use `ItemType`?
Also nit: could you add a comment for it?

Reply:
`ItemType` is intended to be used only by `CredentialStore` but I exported it in case it's useful for others (probably for tests) so that they aren't making their own hard-coded strings for "credentials".

I'll add a comment.

Reply:
Oh! I realize now why you asked about this. 🤦‍♀  This is a line that belongs to the other PR #175. Let me remove it from this PR.